R/logRatioPlot.R

```diff
@@ -6,15 +6,15 @@
 #' Outputs a ggplot2 object faceted by the facetColname or a list of individual
 #' ggplots, one for each facetColname value (typically gene).
 #'
-#' @param data A tidy dataframe of data to plot (Required) (see ?tidyContrasts).
+#' @param contrastsDF A tidy dataframe of data to plot (Required) (see ?tidyContrasts).
```

Comment:
@kar-agg-gen Is there a difference between a `contrastDF` singular and `contrastsDF` plural, used here? I think we want to be consistent and just use one of these (probably the singular `contrastDF`.)

Reply:
Actually Yes **contrastsDF** is output from tidyContrast() . It's basically merged contrast data. As it contains information for all the contrasts, so contrastsDF seems better name.